configure.ac

```diff
@@ -3247,6 +3247,93 @@ case "$host" in
 		;;
 esac
 
+PAL_SUPPORTED=no
+PAL_ARCH=
+PAL_OS=
+
+case "$host" in
+  i*86-*-*)
+	case $host_os in
+	  linux*)
```

Comment:
the script to build System.Native (https://github.com/dotnet/corefx/blob/master/src/Native/build-native.sh) doesn't have a specific parameter to build for android. Would you be ok to simply pass `linux` in the case of android, or should we add this option to the build script?